### PR TITLE
Bugfixes and a template for future development

### LINKED ISSUE #8: d.poll with a topic but no option will result in an empty embed
Expected outcome:
The bot will show the correct usage of the command,
image

Current outcome:
The bot shows an empty embed,
Screenshot from 2020-10-22 22-11-25

### PR DESCRIPTION
**Pull request by medjedqt**
- Helper class for future development improvement
- Poll bugfixes (Including Issue #8)
- Configurable prefix (does not affect prefixes in strings)

### CODE REVIEW

File: src/main/java/RedBot/Main.java [hunk 1]
```diff
@@ -47,9 +47,9 @@ public void onMessageReceived(MessageReceivedEvent event){
         String user = event.getAuthor().getAsTag();
         String userID = event.getAuthor().getId();
         String message = event.getMessage().getContentRaw();
-        if(message.startsWith("d."))
-            bot.exec(event, user, userID,message.split(" ")[0].substring(2), message.split(" "));
-        if(message.contains("<@!") && !message.startsWith("d.") && !message.startsWith(">"))
+        if(message.startsWith(prefix))
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Main.java:50]: Is `prefix` initialised?

File: src/main/java/RedBot/Helper.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            } else if (isInQuotes) {
+                buffer.append(i);
+                buffer.append(" ");
+            } else {str.add(i);} //Place the rest of the strings in the list as is
+        }
+        String[] strarray = new String[str.size()];
+        strarray = str.toArray(strarray);
+        return strarray;
+    }
+}
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Helper.java:32]: Kinda useless right now, but might be usable in the future. Will keep for now.

**Review by LukeHuckman — APPROVED** [review]

File: src/main/java/RedBot/Helper.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import java.util.ArrayList;
+import java.util.List;
+
+public class Helper { //a helper class that contains QoL improvements
+    public static String[] commandParser (String[] message) { //Combines arguments in quotes
+        List<String> str = new ArrayList<String>(); //List of strings to add to
+        Boolean isInQuotes = false; //A flag to decide whether to combine or not
+        StringBuffer buffer = new StringBuffer(); //A buffer to place the strings in
+        for (String i : message) { //Iterate through the message to find any strings in quotes
+            if (message[0]==i) {continue;}
```
Comment (LukeHuckman) [inline src/main/java/RedBot/Helper.java:12]: It's better to use the `equals()` or `equalsIgnoreCase()` method if you're dealing with Strings.

Comment (medjedqt) [reply src/main/java/RedBot/Helper.java:12]: Will keep in mind

**Pull request merged**